Stb (#3823)
mlpack/mlpack: doc/user/install.md

| `-DBUILD_TESTS=ON` | Build `mlpack_test`. | `OFF` |
| `-DUSE_OPENMP=ON` | Use OpenMP for parallelization. | `ON` |
| `-DUSE_PRECOMPILED_HEADERS=OFF` | Disable precompiled headers during build. | `OFF` |

> | `-DUSE_SYSTEM_STB=OFF` | Use version of STB bundled with mlpack. if decided otherwise please be sure to have `stb_image.h`, `stb_image_write.h`, and `stb_image_resize2.h` on the system. | `OFF` |

Reviewer: ```suggestion
| `-DUSE_SYSTEM_STB=OFF` | Use version of STB bundled with mlpack. If set to `ON` make sure `stb_image.h`, `stb_image_write.h`, and `stb_image_resize2.h` are available. | `OFF` |
```

Up to you, I tried to make it a little shorter.
